src/storage/core.rs

```diff
@@ -903,27 +903,32 @@ where
             .iter_mut()
             .map(|b| b.mark_all_as_deleted(key))
             .collect::<FuturesUnordered<_>>();
-        let total = entries_closed_blobs
-            .filter_map(|result| match result {
-                Ok(count) => count,
-                Err(error) => {
-                    warn!("failed to delete records: {}", error);
-                    None
-                }
-            })
-            .fold(0, |a, b| a + b)
+        let total =
+            futures::stream::StreamExt::count(entries_closed_blobs.filter_map(
```

Comment:
Why is the call to `count` in this form? Why not `.count()` at the end of expression?

Reply:
Ambiguity with other imported methods